source/wp-content/themes/wporg-developer-2023/src/handbook-pagination/view.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Add screen reader text to handbook pagination links.
+ * This is needed because the post-navigation-link block cannot accept rich content via the label attribute.
```

Comment:
I tried passing this content through the label attribute. Passing the HTML string in directly as `<span aria-hidden="true">Next </span><span class="screen-reader-text">Next chapter: </span>` worked, but using `wp_kses_post` didn't, I think because it broke the block markup. The rendered block was missing all the passed attributes.

```php
wp_kses_post(__(<span aria-hidden="true">${ visualLabel }</span><span class="screen-reader-text">${ screenReaderText }</span>)
```